sdk/communication/Azure.Communication.CallAutomation/tests.yml

```diff
@@ -5,19 +5,18 @@ extends:
   parameters:
     ServiceDirectory: communication
     Project: Azure.Communication.CallAutomation
+    UseFederatedAuth: true
     CloudConfig:
       Public:
+        ServiceConnection: azure-sdk-tests
+        SubscriptionConfigurationFilePaths:
+          - eng/common/TestResources/sub-config/AzurePublicMsft.json   
         SubscriptionConfigurations:
-          - $(sub-config-azure-cloud-test-resources)
           - $(sub-config-communication-services-cloud-test-resources-common)
           - $(sub-config-communication-services-cloud-test-resources-net)
-      Int:
-        SubscriptionConfigurations:
-          - $(sub-config-communication-int-test-resources-common)
-          - $(sub-config-communication-int-test-resources-net)
     Clouds: Public
     TestResourceDirectories:
```

Comment:
If this is just the root now you can remove this entire setting as that is the default.

Reply:
Also, how were these test deployments working before this change?

Reply:
I dont think it was for some of the other ACS SDKs. We fixed this as part of this work.